recipes/leg2cheb/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "leg2cheb" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/Legendre-to-Chebyshev/archive/{{ version }}.tar.gz
+  sha256: 3ff70cecdb98cb851dc8b04709c616cdd6b3b681dee83c0c8b022a6422718189
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: 
+    - cd src
+    - meson setup build ${MESON_ARGS}
```

This should show you better what the error is. I can highly recommend moving this to a `build.sh` file, though.
```suggestion
    - meson setup build ${MESON_ARGS} || (cat $SRC_DIR/src/build/meson-logs/meson-log.txt; exit 1)
```

---

Thanks @xhochy, I'll try that.